tools/ci_build/github/azure-pipelines/templates/py-packaging-stage.yml

```diff
@@ -238,8 +234,6 @@ stages:
             python.version: '3.7'
           Python38:
             python.version: '3.8'
```

Comment:
same as above

Reply:
It was added long ago. In the past a few releases, we have almost full support for python 3.8 except lacking Windows GPU package and ARM package.